src/App.css

```diff
@@ -2,3 +2,17 @@
   size: 14px;
```

Comment:
this should be `font-size`. Tip to find out if it works, when you double check your work, do it using the inspector, it'll tell you if it doesn't work and why
![image](https://github.com/user-attachments/assets/710f9870-1eea-438d-b6c6-b3add44e3449)